wemake_python_styleguide/violations/consistency.py

```diff
@@ -425,3 +427,35 @@ class BadNumberSuffixViolation(TokenizeViolation):
     #: Error message shown to the user.
     error_template = 'Found underscored number: {0}'
     code = 310
+
+
+class MultipleInComparisonViolation(ASTViolation):
+    """
+    Forbids comparision where multiple 'in's are userd in a statement.
+
+    Reasoning:
+        This is unreadable. Use different comparisons for it.
+        Bring a consistency to the comparison!
+
+    Solution:
+        Refactor your comparison expression.
+        Use different Comparisons.
+
+    Example::
+
+        # Correct:
+        if item in list:
```

Comment:
That's not quite correct. The correct form of ``if x_coord in line in square:`` is ``if x_coord in line and line in square:``

Reply:
Made the change.